scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
> I think we're spending too much effort on designing a perfect logging format, when we could work on the scrapers themselves, e.g. getting them into production :)

Agreed, make it as pretty as possible now, and when one of us gets a creative bout of OCD we can improve it with a custom formatter.